### DISCUSSION TITLE
Errors Testing Audactive on jen.studio

### DISCUSSION THREAD

**Discussion by noobishme** [RonenR/Audactive#12] [Q&A] [answered]
Ronen, Hi! I like where you're going with this. I just created a document for testing (you can see it at https://audactive.com/documents?doc=f7KjRW6hdsnfHyda10eR&to=hlPTFGMNgiQybVbI4mMRvt4JIQg2)

The embed on my site at https://jen.studio/audactive-learning-tool/ is throwing multiple errors as you can see in the console.

**Comment by noobishme** [comment]
Also, as you can also see, the iFrame is too wide for the iFrame. If you change the hero in the CSS to 100% instead of 100vw (or remove it altogether) then it removes the horizontal scrollbar and properly sizes the contents in the iFrame.

hero {
  display: flex;
  flex-direction: column;
  min-height: 100vh;
  width: 100%;
  background-color: antiquewhite;
}

**Reply by RonenR** [reply]
Thanks - this I could not see / reproduce. But - I will look into your suggestion nevertheless. Thank you!

**Comment by RonenR** [comment, marked as answer]
Now fixed. Thank you!

Previous answer:
Indeed - I see that. Thank you so much! I looked at it - seems to me it's a bug I introduced when making it SCORM exportable. I will fix... Thanks again - and nice to e-meet you!

**Reply by RonenR** [reply]
Fixed now. Thank you!!

**Reply by noobishme** [reply]
I've cleared my caches (sites and browser) and the problem is still there...

https://jen.studio/audactive-learning-tool/

![Screenshot 2022-06-23 190610](https://user-images.githubusercontent.com/106134379/175439726-eb7f2099-004e-4d41-9f8b-2b6b1db134df.png)

**Comment by noobishme** [comment]
Thanks @RonenR -- Could you please remind or advise me how to embed the Audactive docs? I thought I remembered a paste-able iFrame code which now I cannot seem to find!

**Reply by RonenR** [reply]
Yes. 
It's simply:
<iframe src="https://audactive.com/documents?doc=f7KjRW6hdsnfHyda10eR" width="100%" height="100%"></iframe>
You can also get it when viewing your document as the author ie via https://audactive.com/author/
